test/jasmine/tests/plots_test.js

```diff
@@ -54,7 +54,7 @@ describe('Test Plots', function() {
             expect(gd._fullData[1].z).toBe(newData[1].z);
             expect(gd._fullData[1]._empties).toBe(oldFullData[1]._empties);
             expect(gd._fullLayout.scene._scene).toBe(oldFullLayout.scene._scene);
-            expect(gd._fullLayout._plots).toBe(oldFullLayout._plots);
+            expect(gd._fullLayout._plots.plot).toBe(oldFullLayout._plots.plot);
```

Comment:
`_plots` is no longer relinked in `Plots.supplyDefaults` as

```js
newFullLayout._plots = oldFullLayout._plots
```

but as

```js
    var oldSubplots = oldFullLayout._plots || {},
        newSubplots = newFullLayout._plots = {};

    var ids = Plotly.Axes.getSubplots(mockGd);

    for(var i = 0; i < ids.length; i++) {
        var id = ids[i],
            oldSubplot = oldSubplots[id],
            plotinfo;

        if(oldSubplot) {
            plotinfo = newSubplots[id] = oldSubplot;
        }
        else {
            plotinfo = newSubplots[id] = {};
            plotinfo.id = id;
        }

        plotinfo.x = getAxisFunc(id, 'x');
        plotinfo.y = getAxisFunc(id, 'y');
        plotinfo.xaxis = plotinfo.x();
        plotinfo.yaxis = plotinfo.y();
    }
```

cc @rreusser